### PR TITLE
Added usage examples for multiple API symbols

### PR DESCRIPTION
**Pull request by Jake-Short**
Added usage examples for image.resize_with_crop_or_pad, image.resize_with_pad, image.random_brightness, image.random_contrast, image.random_crop, math.multiply_no_nan, and math.reciprocal_no_nan.

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -1144,6 +1144,14 @@ def multiply_no_nan(x, y, name=None):
 
   Returns:
     The element-wise value of the x times y.
+
+  Usage Example:
```
Comment (Ayush517) [inline tensorflow/python/ops/math_ops.py]: "Usage Example" is to be added before "Args" and "Returns" blocks as is done in the entire file. Please shift the code accordingly.

**Review by F20170604 — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py (hunk 1 above)
Comment (Ayush517) [reply tensorflow/python/ops/math_ops.py]: @Jake-Short Please do the same for all the usage examples you've added.

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -1510,14 +1517,7 @@ def resize_image_with_pad_v2(image,
...
     If `images` was 3-D, a 3-D float Tensor of shape
-    `[new_height, new_width, channels]`.
-	
-  Usage Example:
-    ```python
-    >> import tensorflow as tf
-    >> x = tf.random.normal(shape=(256, 256, 3))
-    >> tf.image.resize_with_pad(x, 200, 200, tf.image.ResizeMethod.BILINEAR, False)
-    ```
+    `[new_height, new_width, channels]`
```
Comment (Ayush517) [inline tensorflow/python/ops/image_ops_impl.py]: Please add the full stop as was in the original code.

`[new_height, new_width, channels]`.

Comment (Ayush517) [inline tensorflow/python/ops/image_ops_impl.py]: Please add the full stop as was in the original code.

`[new_height, new_width, channels]`.

**Review by F20170604 — DISMISSED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 2 above)
Comment (Jake-Short) [reply tensorflow/python/ops/image_ops_impl.py]: Sorry about that, I have resolved that issue.

**Comment by Ayush517** [comment]
@Jake-Short Looks good to me now 👍

**Comment by nikochiko** [comment]
@Ayush517 , I think it would be better to use the triple left-angle bracket (`>>>`) syntax for examples as recommended in the [contributing guidelines](https://www.tensorflow.org/community/contribute/docs_ref#testable_docstrings). It would also make the examples compatible with `DocTest`. For testing the return values, Google Colab notebooks with `TensorFlow 2` can be used.

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -939,6 +939,13 @@ def resize_image_with_crop_or_pad(image, target_height, target_width):
   If `width` or `height` is smaller than the specified `target_width` or
   `target_height` respectively, this op centrally pads with 0 along that
   dimension.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> x = tf.random.normal(shape=(256, 256, 3))
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: If you make a fake 4x4 image and resize it to be a 2x2 image I think you can use nice round numbers and have a testable example

File: tensorflow/python/ops/math_ops.py [hunk 4]
```diff
@@ -1136,6 +1136,14 @@ def div_no_nan(x, y, name=None):
 @dispatch.add_dispatch_support
 def multiply_no_nan(x, y, name=None):
   """Computes the product of x and y and returns 0 if the y is zero, even if x is NaN or infinite.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
+    >> x = tf.constant([[1.0, 3.0], [5.0, 4.0]])
+    >> y = tf.constant([[7.0, 2.0], [1.0, 9.0]])
+    >> tf.math.multiply_no_nan(x, y) # [[7, 6] [5, 36]]
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: You don't need the tf.constants, just pass the python lists of values and let tf convert

Comment (alextp) [reply tensorflow/python/ops/math_ops.py]: though actually passing NaN in a few entries and showing the result would be useful

File: tensorflow/python/ops/image_ops_impl.py [hunk 5]
```diff
@@ -939,6 +939,13 @@ def resize_image_with_crop_or_pad(image, target_height, target_width):
   If `width` or `height` is smaller than the specified `target_width` or
   `target_height` respectively, this op centrally pads with 0 along that
   dimension.
+  
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: no need to import tensorflow as tf in every docstring

**Review by alextp — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@Jake-Short  thank you, it is still failing doctest can you please check here for [logs](https://source.cloud.google.com/results/invocations/a9741221-910e-457d-9c81-7c18b00dca03/targets/%2F%2Ftensorflow%2Ftools%2Fdocs:tf_doctest/tests).
Please run the doctest locally as mentioned here in the [contributor guidelines](https://www.tensorflow.org/community/contribute/docs_ref).

Also, can you please help to fix Ubuntu Sanity errors? Thanks!

**Comment by Jake-Short** [comment]
It looks like a lot of these have been added in later pull requests (I.E. #34798 and #35332). I will close this PR in favor of those.

**Pull request closed without merging**